lib/mrsk/utils/healthcheck_poller.rb

```diff
@@ -0,0 +1,39 @@
+class Mrsk::Utils::HealthcheckPoller
```

Comment:
Named this `HealthcheckPoller` rather than something like `Healthcheck`, since the actual healthcheck is provided for us by Docker. This class just needs to know how to wait for the status to reach something runnable.